core-developers/become-core-developer.rst

```diff
@@ -60,6 +60,12 @@ are granted through these steps:
      were in the form of a separate post on the already open topic with
      the poll.
 
+Getting a python.org Email Address
+----------------------------------
```

Comment:
Use "sentence case" for headings:

https://devguide.python.org/documentation/style-guide/#capitalization


```suggestion
Getting a python.org email address
----------------------------------
```